ipc/api/src/lib.rs

```diff
@@ -31,3 +33,30 @@ pub fn ethers_address_to_fil_address(addr: &ethers::types::Address) -> anyhow::R
     let eth_addr = EthAddress::from_str(&raw_addr)?;
     Ok(Address::from(eth_addr))
 }
+
+/// Marker type for human readable form.
+pub struct HumanReadable;
```

Comment:
The existing `HumanReadable` implementations only apply the string format [if the serializer is human readable](https://github.com/consensus-shipyard/ipc/blob/9fe1ee01e85ef430b77aceccc90a934ec947edee/fendermint/vm/encoding/src/lib.rs#L36-L39).

I reckon you should come up with a different name because this is misleading.

Reply:
Changed name to `AsString`.